ecs-cli/modules/commands/local/local_command.go

```diff
@@ -164,33 +164,33 @@ func flagName(longName string) string {
 func flagDescription(longName, cmdName string) string {
 	m := map[string]map[string]string{
 		flags.TaskDefinitionCompose: {
-			upCmdName: "The Compose file `name` of a task definition to run.",
+			upCmdName: "Specifies the filename `value` that contains the Docker Compose content to run locally.",
 		},
 		flags.TaskDefinitionFile: {
-			createCmdName: fmt.Sprintf("The file `name` of a task definition json to convert. If not specified, defaults to %s", project.LocalInFileName),
-			upCmdName:     fmt.Sprintf("The file `name` of a task definition json to convert and run. If not specified, defaults to %s", project.LocalInFileName),
-			psCmdName:     "List all running containers matching the task definition file `name`.",
-			downCmdName:   "Stop and remove all running containers matching the task definition file `name`.",
+			createCmdName: fmt.Sprintf("Specifies the filename `value` that contains the task definition JSON to convert to a Docker Compose file. If one is not specified, the ECS CLI will look for %s.", project.LocalInFileName),
+			upCmdName:     fmt.Sprintf("Specifies the filename `value` containing the task definition JSON to convert and run locally.  If one is not specified, the ECS CLI will look for %s.", project.LocalInFileName),
+			psCmdName:     "Lists all running containers matching the task definition filename `value`.",
+			downCmdName:   "Stops and removes all running containers matching the task definition filename `value`.",
```

Comment:
do we need to mention that this removes the local network as well if there are no more containers? if so should we mention that `up` creates one (if it doesn't alreayd exist)?

Reply:
Hm. That's a good point. I'm leaning towards not including it just because I'm not sure if that'd affect how you'd use these commands.

But! I think it'd make sense to add the fact that we spin up a network in the Usage section? 

> Run containers locally from an ECS Task Definition. NOTE: Creates a docker-compose file in current directory.

> Run containers locally from an ECS Task Definition. NOTE: Creates a docker-compose file in current directory and creates an ecs-local-network if one doesn't exist.

How about that?